modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java

```diff
@@ -922,7 +922,9 @@ private String _normalizeFriendlyURL(String friendlyURL) {
 		if (Validator.isNotNull(friendlyURL) &&
 			friendlyURL.startsWith(StringPool.SLASH)) {
 
-			return friendlyURL.substring(1);
+			String normalizedFriendlyURL = friendlyURL.substring(1);
+
+			return StringUtil.toLowerCase(normalizedFriendlyURL);
 		}
 
 		return friendlyURL;
```

Comment:
We are only lowercasing on the case of the '/friendlyURL' 
Should we do it in both cases or in the case that the 'friendlyURL' do not begin with the '/' character won't be necessary or it will never happens?

Reply:
do it worth to add tests to FriendlyURLServletTest with this case?

Reply:
On another hand, this case fails:

- Add a page with firendlyURL "/testpagé"
- Add a new redirection under configuration -> redirection with a Source URL "/testpagé" and a Destination URL like "https://www.google.com/"
- Try to access /TESTPAGÉ

Can you take a look?

Reply:
Hi @AliciaGarciaGarcia ,
> We are only lowercasing on the case of the '/friendlyURL' Should we do it in both cases or in the case that the 'friendlyURL' do not begin with the '/' character won't be necessary or it will never happens?

While I was testing it I did not manage to create a use case where the layoutFriendlyURL did not start with a "/" but I can modify the solution in order to handle both cases.


> do it worth to add tests to FriendlyURLServletTest with this case?
I don't think but Brian might request one when the solution is forwarded.


> On another hand, this case fails:
> 
> * Add a page with firendlyURL "/testpagé"
> * Add a new redirection under configuration -> redirection with a Source URL "/testpagé" and a Destination URL like "https://www.google.com/"
> * Try to access /TESTPAGÉ
> 
> Can you take a look?

Thanks! Looking into this case.

Reply:
> > do it worth to add tests to FriendlyURLServletTest with this case?
> 
> I don't think but Brian might request one when the solution is forwarded.

Best to do it on this PR I think